packages/app-marketplace-admin/cdk-stack.ts

```diff
@@ -0,0 +1,31 @@
+import { execSync } from 'child_process'
+import { createBaseStack, createSite } from '@reapit/ts-scripts/src/cdk'
+import { join } from 'path'
+import config from './config.json'
+
+const createStack = async () => {
+  const stack = createBaseStack({
+    namespace: 'cloud',
+    appName: 'app-marketplace-admin',
+    component: 'site',
+    accountId: '028446965111',
```

Comment:
you shouldddd be able to get accountId and region from the account you're logged in with

Reply:
it didn't get it from process.env unfortunately so had to add it manually :(

Reply:
that's weird, it works in app-builder-backend :/ is accountId in config.json already? if so it should be pulled from there rather than hard coded